### DISCUSSION TITLE
Hot to change hover behaviour of a ButtonElement?

### DISCUSSION THREAD

**Discussion by felixwoestmann** [vueform/vueform#356] [Questions]
Hello,
I've implemented a simple form in my application.
Currently when I hover over the submit button element it increases in size due to the class: ` hover:scale-105`. I would like to change this behavior so the button rather changes colors. How can I achieve this? I tried the `replace-class` attribute but didn't reach my goal.

**Comment by adamberecz** [comment]
Would this work on `ButtonElement`? `:replace-class="{ button_enabled: { 'hover:scale-105': 'bg-red-500' } }"`
